### PR TITLE
[DOCS-497]update-whats-new-pages

### PR DESCRIPTION
**Pull request by stef-cobaltio**
## Changelog
Updates to page layout for all existing release notes to remove the pages subtitle (description) and update the pageinfo boxes copy.

Added new images and new content to the August 2024 release notes.

Added pages and updated their listing hiearchy to house the release notes for the months of September and October.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Comment by stef-cobaltio** [comment]
Updating the pull request to have an attached ticket, DOCS-497

File: content/en/Product Updates/release-notes-august-2024.md [hunk 1]
```diff
@@ -37,3 +36,35 @@ The problem we identified was that targets were used in different parts of the a
...
+
+---
+
+#### Enhanced SAML Configuration Page in Org Settings
+
+Customers utilizing Service Provider initiated SAML SSO can now seamlessly access this feature directly within the platform. After configuring SAML settings, organizations will now display a Cobalt Sign In URL. 
+
+This URL simplifies the user experience by automatically redirecting users to their designated SAML identity provider. For convenient access, users relying on SAML SSO for sign-in can now bookmark this link and effortlessly sign in to Cobalt. 
+
+![Create Asset - Add Associated Targets](/release-notes/saml-updates.png "SAML Updates")
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-august-2024.md:48]: Just wanted to make sure the IdP details (URL with trexcloud.com) used in the image are fake.

I would also update the name of each image in brackets to match the name in quotation marks e.g. [SAML Updates]..."SAML Updates"

File: content/en/Product Updates/release-notes-june-2024.md [hunk 2]
```diff
@@ -3,11 +3,10 @@ title: "Release Notes: June 2024"
 linkTitle: "June 2024"
 weight: 46
 description: >-
-     Explore What's New from Cobalt This Month
 ---
 
 {{% pageinfo %}} 
-Stay Up-to-Date with the Latest Enhancements on the Cobalt Platform. 
+Explorew What's New from Cobalt This Month
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-june-2024.md:9]: Update spelling of 'Explore'

**Comment by mtsuji-cobalt** [comment]
Some other notes @stef-cobaltio:

• Are October + September pages supposed to have content? they are both blank
• I see certain boxes updated like this but not all are in this style:
Screenshot 2024-10-31 at 13 22 09
i think we'd want to update all of these across the entire ProDocs at once, which i think we might need engineering's help with

**Comment by stef-cobaltio** [comment]
@mtsuji-cobalt  

- October and September are blank right now, wanted to get them published first as part of this PR.
- For the updated removal of description and pageinfo. I am OK pushing this change as we have updated all the descriptions/pageinfos across the release notes for consistency in this section. I have it on my list to handle sitewide next week.

**Comment by stef-cobaltio** [comment]
@mtsuji-cobalt I updated the image and had to turn the PR back into a draft - can you still continue reviewing it or do I need to put it back into another state?

File: content/en/Product Updates/release-notes-august-2024.md (hunk 1 above)
Comment (stef-cobaltio) [reply content/en/Product Updates/release-notes-august-2024.md]: Updated the image to blur the IDp SSO string.

File: content/en/Product Updates/release-notes-august-2024.md [hunk 3]
```diff
@@ -37,3 +36,35 @@ The problem we identified was that targets were used in different parts of the a
...
+
+---
+
+#### Enhanced SAML Configuration Page in Org Settings
+
+Customers utilizing Service Provider initiated SAML SSO can now seamlessly access this feature directly within the platform. After configuring SAML settings, organizations will now display a Cobalt Sign In URL. 
+
+This URL simplifies the user experience by automatically redirecting users to their designated SAML identity provider. For convenient access, users relying on SAML SSO for sign-in can now bookmark this link and effortlessly sign in to Cobalt. 
+
+![Create Asset - SAML Updates](/release-notes/saml-updates.png "SAML Updates")
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-august-2024.md:48]: For lines 48, 60, and 70 - i would remove 'Create Asset -' from the image description/title in the brackets []

File: content/en/Product Updates/release-notes-april-2024.md [hunk 4]
```diff
@@ -3,11 +3,10 @@ title: "Release Notes: April 2024"
 linkTitle: "April 2024"
 weight: 48
 description: >-
-     Explore What's New from Cobalt This Month
 ---
 
 {{% pageinfo %}} 
-Stay Up-to-Date with the Latest Enhancements on the Cobalt Platform. 
+     Explore What's New from Cobalt This Month
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-april-2024.md:9]: I see that the blue info box is updated only for April, August, and September 2024. I wasn’t sure which style was intended or you intended to just remove the subtitle and descriptions.
Screenshot 2024-11-01 at 11 48 33

Screenshot 2024-11-01 at 11 48 37

File: static/release-notes/saml-updates (1).png
Comment (mtsuji-cobalt) [inline static/release-notes/saml-updates (1).png:1]: The blurred image hasn’t been updated in the code I think - you’d have to upload it with the same file name as before ‘saml-updates.png’ (the one you have here is ‘saml-updates(1).png’)

File: content/en/Product Updates/release-notes-september-2024.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+#### Updated Pentest Attestation Letters
+
+We revamped the attestation letter's style for better PDF viewing and downloading, aligning it with our current brand guidelines.
+
+{{% pageinfo color="info" %}}
+View more scan controls available [here](https://docs.cobalt.io/platform-deep-dive/scans/scans/).
+{{% /pageinfo %}}
+
+![Attestation Letter Improvements](/release-notes/attestation-letter.png “Attestation “Letter Improvements)
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-september-2024.md:22]: All 3 images here need to adjust/close the quotation marks at the end around the image description e.g. "Attestation Letter Improvements"

File: content/en/Product Updates/release-notes-september-2024.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+     Explore What's New from Cobalt This Month
+{{% /pageinfo %}}
+
+### September 2024
+
+#### Updated Pentest Attestation Letters
+
+We revamped the attestation letter's style for better PDF viewing and downloading, aligning it with our current brand guidelines.
+
+{{% pageinfo color="info" %}}
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-september-2024.md:18]: We can remove this part, it relates to Scan info from the previous month's notes.

File: content/en/Product Updates/release-notes-april-2024.md (hunk 4 above)
Comment (stef-cobaltio) [reply content/en/Product Updates/release-notes-april-2024.md:9]: @mtsuji-cobalt The solid blue box is definitely what I am going for, I think when the text was indented it started to create that inner white box thing. I just went through and updated all the formats so this should be resolved.

**Review by mtsuji-cobalt — APPROVED** [review]

**Pull request merged**